server/logging.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"testing"
+
+	"github.com/mattermost/logr"
+)
+
+type field struct {
+	name string
+	val  interface{}
+}
+
+// TestingTarget is a Logr target proxies logs through a stdlib testing interface.
+// This allows tests that spin up App instances to avoid spewing logs unless the test fails or -verbose is specified.
+type TestingTarget struct {
```

Comment:
@agnivade Is this testing.TB based target even needed?  It came from a comment in the ticket, but since then you've indicated you want to log directly into a bytes.Buffer.

Reply:
Oh I see. Yeah it's not needed then. We can get rid of it.

Reply:
Ok, it's removed.